contextualhelp/data.json

```diff
@@ -1109,7 +1195,7 @@
         ]
       },
       {
-        "route_name": "MESSAGES_HOME",
+        "route_name": "MESSAGES_INBOX",
```

Comment:
ditto

Reply:
It should be verified what is the current behaviour of messages home since we implemented the internal tabs (same should be checked on services)
cc. @Vangaorth @adelloste

Reply:
Specs are as following:
- FAQ for **received** messages have to use "**MESSAGES_INBOX**"
- FAQ for **archived** messages have to use "**MESSAGES_ARCHIVE**"
- FAQ for **national** services have to use "**SERVICES_NATIONAL**"
- FAQ for **local** messages have to use "**SERVICES_LOCAL**"